modules/passkey/hardhat.config.ts

```diff
@@ -1,4 +1,5 @@
 import '@nomicfoundation/hardhat-toolbox'
+import '@nomicfoundation/hardhat-ethers'
```

Comment:
Somehow not adding this package explicitly to the `package.json` and importing here results in a type mess for `ethers.provider`